templates/github-action-test.yml

```diff
@@ -25,10 +25,10 @@ jobs:
 {% endif %}
     steps:
     - uses: actions/checkout@v2
-    - name: Use Node.js 12.x
-      uses: actions/setup-node@v1
+    - name: Use Node.js 16.x
```

Comment:
A bunch of the apps had been rolled to v16 in their test actions, so I made the change here as well. v12 is old!